### DISCUSSION TITLE
网站404

### DISCUSSION THREAD

**Discussion by deleteDB** [1Panel-dev/1Panel#6373] [Q&A]
### 1Panel 版本

v1.10.14-lts

### 问题描述

使用php8 + ip端口的方式创建网站 可以访问 后续新增域名 新增的域名无法访问显示404文件确定存在  ip + 端口的方式依旧正常

按照上面的步骤创建的时候直接使用域名就正常

### 重现步骤

创建网站的时候使用 `运行环境` 环境为PHP8  主域名为 ip:端口 的方式 可以访问网站 在创建之后  在  配置 ->  域名设置 -> 新增域名 增加一个域名之后就无法访问(dns已生效) 就会显示404(文件确定存在) 使用创建时使用的 ip + 端口就可以访问 这边尝试使用域名重新创建相同的配置就可以访问 感觉就像有什么配置没有修改到位 之前版本就出现了 我以为是我修改配置造成的  今天从新安装一台服务器发现还是一样

**Comment by zhengkunwang223** [comment]
IP 为域名的网站 可以现设置为默认网站 然后看一下会不会有这个问题

**Comment by deleteDB** [comment]
> IP 为域名的网站 可以现设置为默认网站 然后看一下会不会有这个问题

尝试设置为默认网站还是无法访问404 把域名更换为创建时的 ip+端口 路径不变可以正常访问

**Comment by zhengkunwang223** [comment]
> > IP 为域名的网站 可以现设置为默认网站 然后看一下会不会有这个问题
> 
> 尝试设置为默认网站还是无法访问404 把域名更换为创建时的 ip+端口 路径不变可以正常访问

![image](https://github.com/user-attachments/assets/e87f8e33-7c86-4169-8f9e-6e73c14c5f2d)
 加我企业微信 我远程看一下
